test/parallel/test-http-url.parse-https.request.js

```diff
@@ -31,8 +32,8 @@ const fs = require('fs');
 
 // https options
 const httpsOptions = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fs.readFileSync(`${fixturesDir}/keys/agent1-key.pem`),
```

Comment:
This could be using method `fixtures.readKey` instead of `fs.readFileSync` to get the keys.

Reply:
Good call.  I followed the distributed Code & Learn task description pretty literally, so this thought never occurred to me. Thanks for the tip. :+1:

Reply:
P.S. Updated with another commit.